gulp/FabricBuild.js

```diff
@@ -17,6 +40,33 @@ gulp.task('Fabric-nuke', function () {
 });
 
 //
+// Style Linting
+// ---------------------------------------------------------------------------
+gulp.task('Fabric-styleHinting',  function() {
+    if(Config.buildSass) {
+        var stream;
+        stream = gulp.src(Config.paths.srcSass + '/Fabric.scss')
+            .pipe(Plugins.gulpif(Config.debugMode, Plugins.debug({
+                title: "Checking SASS Compile errors and linting"
+            })))
+            .pipe(Plugins.scsslint())
+            .pipe(ErrorHandling.SASSlintErrors());
+      
+    } else {
+        stream = gulp.src(Config.paths.srcLess + '/Fabric.less')
+            .pipe(Plugins.gulpif( Config.debugMode, Plugins.debug({
```

Comment:
There's an extra space after the 'gulpif(' here but not on line 49 -- not sure what your pattern is for gulpif, but it could probably be more consistent.